### DISCUSSION TITLE
May I ask if it is possible to include GaussDB on the page at https://dbeaver.io/about/, thank you~

### DISCUSSION THREAD

**Discussion by GaussDB-Fans** [dbeaver/dbeaver#36429] [Q&A] [answered]
RT: May I ask if it is possible to include GaussDB on the page at https://dbeaver.io/about/, thank you~
@E1izabeth @serge-rider

**Comment by E1izabeth** [comment, marked as answer]
Hi, @GaussDB-Fans!
We have included GaussDB in this list.
Thanks
